src/blocks/avishaan24/medium.css

```diff
@@ -0,0 +1,190 @@
+.container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 400px;
+    height: 49vh;
+    border: 2px solid rgb(247, 242, 242);
+    background-color: #dee2e6;
+    box-shadow: 5px 10px 8px 2px #888888;
+    border-radius: 18px;
+}
+h2{
+    text-align: center;
+    margin: 2vh 0px 5px 0px;
+}
+#box{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 51%;
+    height: 49vh;
+    margin: 10px auto;
+}
+.button{
+    padding: 0px;
+    margin: 2px 2px;
+    border: 2px solid black;
+    border-radius: 6px;
+    cursor: pointer;
+    color: white;
+    background-color:darkslategrey;
+    width: 48px;
+    height: 8vh;
+    text-align: centre center;
+    box-shadow: inset 8px 8px 8px rgb(119, 156, 226);
+}
+
+
+.input{
+    display: block;
+    margin: 10px auto ;
+    padding: 0px;
+    /* border: 2px solid black; */
+    border-radius: 3px;
+    width: 390px;
+    height: 9vh;
+    text-align: right;
+    font-size:x-large;
+    background: #c6c8c9;
+    box-shadow: inset 8px 8px 8px #7b7b7c;
+}
+.border{
+    margin: 20;
+    border-radius: 12px;
+    border: 3px solid black;
+    width: fit-content;
+}
+#equal{
+    background-color: darkgoldenrod;
+}
+body{
+    margin: 0px;
+    padding: 0px;
+    height: 100vh;
+}
+
+.background{
+    width: 100%;
+    height: 100vh;
+    background-image: url("download\ \(4\).jpg");
```

Comment:
Please provide the license to the images you used, so that we can be sure that we don't violate copyright.

Reply:
i didn't consider copyright i wiil definitely change the image